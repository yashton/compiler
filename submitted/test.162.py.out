(PUNCT "(")
(ID "d")
(PUNCT "=")
(LIT "1.4")
(NEWLINE)
(INDENT)
(PUNCT "+")
(LIT 8)
(PUNCT "}")
(NEWLINE)
(ID "hello")
(ID "there")
(DEDENT)
(ENDMARKER)
