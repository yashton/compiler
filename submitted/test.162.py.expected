(PUNCT "(")
(ID "d")
(PUNCT "=")
(LIT "1.4")
(PUNCT "+")
(LIT 8)
(PUNCT "}")
(NEWLINE)
(INDENT)
(ID "hello")
(ID "there")
(DEDENT)
(ENDMARKER)
